specs/experimental/builder.md

```diff
@@ -0,0 +1,172 @@
+# Builder API
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Sequencer Builder Interaction](#sequencer-builder-interaction)
+  - [Requesting a Block](#requesting-a-block)
+    - [Liveness Failsafe](#liveness-failsafe)
+  - [Mempool Forwarding](#mempool-forwarding)
+  - [Builder Configuration](#builder-configuration)
+- [Structures](#structures)
+  - [`PayloadRequestV1`](#payloadrequestv1)
+  - [`BuilderPayloadV1`](#builderpayloadv1)
+- [Methods](#methods)
+  - [`builder_getPayloadV1`](#builder_getpayloadv1)
+  - [`builder_forwardTransactionV1`](#builder_forwardtransactionv1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+This document provides an in-depth specification for integrating a Builder API within the Optimism Protocol
```

Comment:
small nit: nothing about this proposal enshrines PBS into the protocol but saying "within the Optimism Protocol" makes it sound like ePBS

Reply:
good point, will change to "Optimism Stack". I was a bit confused because `/specs` defines "the plain english specs for Optimism, a minimal optimistic rollup protocol" but this change is actually for the "Optimism Stack".